OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -337,15 +337,20 @@ public interface IRenderAboveShroudWhenSelected
 		bool SpatiallyPartitionable { get; }
 	}
 
+	/// <summary>
+	/// Indicates target types as defined on <see cref="Traits.ITargetable"/> are present in a <see cref="Primitives.BitSet{T}"/>.
+	/// </summary>
+	public sealed class TargetableType { TargetableType() { } }
```

Comment:
Can we rename this to `TargetType` to match previous conventions?

Reply:
Conflicts with the also reasonably named `TargetType` (i.e. the enum that tells you the type of a `Target`).

Reply:
Ah right, of course. I agree that `TargetableType` is our next best option.